image/nitf.ksy

```diff
@@ -0,0 +1,819 @@
+meta:
+  id: nitf
+  title: National Imagery Transmission Format
+  file-extension:
+    - ntf
+    - nitf
+    - ntf.r0
+    - ntf.r1
+    - ntf.r2
+    - ntf.r3
+    - ntf.r4
+    - ntf.r5
+  ks-version: 0.9

```

Comment:
Now that we got rid of all `_sizeof` usages in commit https://github.com/kaitai-io/kaitai_struct_formats/pull/294/commits/fb0333d5d21ca782295f00b571373bce240a9e03, which was the only feature available from the (not yet released) 0.9 version, you can calmly lower the minimum KS compiler version needed to compile your spec to **0.8**.

```suggestion
  ks-version: 0.8
```